src/SmartComponents/SystemAdvisor/SystemAdvisor.js

```diff
@@ -604,7 +605,15 @@ const BaseSystemAdvisor = () => {
     dataFetch();
   }, []);
 
-  return (
+  return inventoryReportFetchStatus === 'fulfilled' &&
+    entity.insights_id !== null &&
+    activeReports.length <= 0 ? (
```

Comment:
How did you come up with this condition?

Reply:
This is also being added/corrected in Compliance[0]. After some discussion, the condition for showing the `NotConnected` component should only be when it does not have a `insights_id` and have application specific empty states otherwise.

[0] https://github.com/RedHatInsights/compliance-frontend/pull/1565

Reply:
This solves it, thank you, Sebastian.